spec/presenters/order_delivery_status_actions_presenter_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'spec_helper'
+
+describe OrderDeliveryStatusActionsPresenter do
+  describe "#render" do
+    let(:user) { double("User") }
+    let(:order) { double("Order") }
+    let(:view_context) { ActionView::Base.new }
+
+    let(:presenter) { OrderDeliveryStatusActionsPresenter.new(user, order, view_context) }
+
+    before do
+      allow_any_instance_of(Order::DeliveryStatusPolicy).to receive(:mark_delivered?) { allow_mark_delivered }
+      allow_any_instance_of(Order::DeliveryStatusPolicy).to receive(:mark_undelivered?) { allow_undo_mark_delivered }
```

Comment:
`allow_any_instance_of` is a bit of an anti-pattern, better to mock in this style:

    allow(Order::DeliveryStatusPolicy).to receive(:new) { policy }
    let(:policy) { double(:mark_all_delivered? => false, :mark_undelivered? => true) }

Reply:
Hmmm I need some help fully grokkin this one...

Reply:
Actually, disregard. I got it. Just had a typo.